test-js/unit/prompting/PromptPanelSpec.js

```diff
@@ -546,14 +546,20 @@ define([ 'dojo/number', 'dojo/i18n', 'common-ui/prompting/PromptPanel',
             }
           });
           comp.type = "ScrollingPromptPanelLayoutComponent";
-          spyOn(window, "$");
+
+          // TODO: Find alternative for:
+          //spyOn(window, "$");
```

Comment:
@dcleao we could have something like:
`var _jqHolder={$:$};
spyOn(_jqHolder, "$")`
ant then the expect would be
`expect(_jqHolder.$).not.toHaveBeenCalled()`